libtenzir/src/argument_parser2.cpp

```diff
@@ -275,6 +291,9 @@ auto argument_parser2::usage() const -> std::string {
           return fmt::format("{}", type_kind::of<data_to_type_t<T>>);
         }
       },
+      [](const setter<located<data>>&) -> std::string {
+        return "constant";
```

Comment:
```suggestion
        return "any";
```

Reply:
So far we have been very consistent with using the same name for runtime values as for constant ones. For example, it's `load_file path:string`, not `load_file path:constant_string` or whatever.

Reply:
Okay, but if the value is not a constant we warn saying "value must be constant". Why not include that in the type description?

Reply:
We do not warn, we even emit an error. The main motivation for not including it in the type back then was because most of the time, the user doesn't care and intuitively does the right thing. So far, this assumption has played out really well, so I don't see a reason to complicate the whole reference right now.

Reply:
I kind of disagree for this case where you just want a constant of any type but I changed it.